aap-common/ref-aap-using-playbooks.adoc

```diff
@@ -2,7 +2,7 @@
 
 = Using the playbooks
 
-The playbooks are explained in different chapters of this documentation but not all of them. 
+The playbooks are explained in different chapters of this documentation but not all of them.
```

Comment:
Only some of the playbooks are described in this document.

Reply:
Fixed